src/screens/list/UserList.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Navbar from "../../components/Navbar";
+import List from "../../components/Navbar";
+
+const UserList = (props) => {
+  return (
+    <div>
+      <Navbar />
+      <div className="table-responsive  ">
+        <h3>Userlist</h3>
+        <table className="table align-middle">
```

Comment:
Add this to the parent component as now you are creeating a new table for every user